pywwt/annotation.py

```diff
@@ -252,6 +255,100 @@ def _on_trait_change(self, changed):
         super(Line, self)._on_trait_change(changed)
 
 
+class FieldOfView():
+    """
+    A collection of polygon annotations. Takes the name of a pre-loaded 
+    telescope and displays its field of view.
+    """
+    # links for kepler. guide: https://keplerscience.arc.nasa.gov/k2-fields.html
+    # footprint: https://raw.githubusercontent.com/KeplerGO/K2FootprintFiles/master/json/k2-footprint.json
+
+    # more efficient method than CircleCollection of changing trait values?
+    def __init__(self, parent, telescope, center, rot, **kwargs):
+        self.parent = parent
+        self.available = self.parent.instruments.available
+        self.dim = self.parent.instruments.dim
+        
+        # list of IDs of annotations created in this FieldofView instance
+        self.active = []
+        self._gen_fov(telescope, center, rot, **kwargs)
+
+    def _gen_fov(self, telescope, center, rot, **kwargs):
+        #telescope = telescope.lower()
+        if telescope not in self.available:
+            raise ValueError('the given telescope\'s field of view is unavailable at this time')
+
+        position = self.dim[telescope][0]
+        dimensions = self.dim[telescope][-1]
+        # dimensions is a list of lists [ [[ras], [decs]], '', ...]
+
+        # test that pos matches what the user entered
+        if center and position == 'absolute':
+            raise ValueError('the given telescope does not take center coordinates')
+        elif not center and position == 'relative':
+            raise ValueError('the given telescope requires center coordinates')
+
+        # draw jwst nircam's inner panels first
+        if telescope[-6:] == 'nircam':
+            self._gen_fov('nircam_short', center, rot, **kwargs)
+
+        for panel in dimensions:
+            ras = (panel[0] * u.deg).value
+            decs = (panel[1] * u.deg).value
+            ra_r, dec_r = self._rotate(ras, decs, rot)
+
+            if telescope == 'k2':
+                ra_tr = ra_r; dec_tr = dec_r
```

Comment:
If you want to write this on one line, I think ``ra_tr, dec_tr = ra_r, dec_r`` would be better.